Adds datetime tooltips to Oldest and Newest buttons

cgorringe opened the pull request:
Implements #475 by adding datetime tooltips to the Oldest and Newest buttons.
Works only when Wayback Count is turned on.

tikhsuP reviewed (commented):
It works well!
I noticed that you also made the change that when on pages that have never been saved, the SPN button doesn't flip.

I have the following suggestions:
- We should update the tooltip timestamps when using the Search URL.
- We should update the tooltip timestamps after a successful save.
- Should we not flip the SPN button when on excluded URLs?

cgorringe commented:
OK, I implemented the following suggestions:
1. Cleared tooltips during Search.
2. Updated timestamps after Save.
3. Didn't change flipping behavior. Keep flip on Excluded URLs for now.
Also did these:
4. Display wayback count when setting checked on.
5. Now showing `0` in badge if valid page that hasn't yet been archived.

tikhsuP reviewed (approved):
Looks good!
Thanks for the changes.
